tests/kani/UninitializedMemory/fixme_main.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// At the moment, Kani is unable to detect the usage of uninitialized memory
+// See https://github.com/model-checking/kani/issues/920
+
+#[kani::proof]
+fn main() {
+    let mut v: Vec<u8> = Vec::with_capacity(8);
+    unsafe {
+        v.set_len(3);
+    }
+    let _b = v[0]; //< reading uninitialized memory
+    assert(false);
```

Comment:
Please add a comment to explain the point of this `assert(false)`.

Reply:
The `assert(false)`, makes no sense to me. This test will fail whether or not we fix the issue. Which seems to me to break the point of a "fixme" test

Reply:
that's true
I don't know how to make that test work then